### DISCUSSION TITLE
Update,Delete

### DISCUSSION THREAD

**Discussion by NimeshPiyumantha** [community/community#21016] [New to GitHub] [answered]
How to update and delete a commit in the git hub repository?

**Comment by FahdSeddik** [comment, marked as answer]
Hey!

I think you might find an answer in this discussion on stackoverflow [here](https://stackoverflow.com/questions/448919/how-can-i-remove-a-commit-on-github#:~:text=First%2C%20remove%20the%20commit%20on,editor%20window%20that%20pops%20up.). Hope this helps!

**Reply by FahdSeddik** [reply]
Make sure to mark it as an answer if it does help, so people would see it easier.
